models/user.py

```diff
@@ -17,6 +16,15 @@ class RoleEnum(Enum):
     STAFF = 3
     ADMIN = 4
 
+    @classmethod
+    def get_values(cls):
```

Comment:
Should this be designated a private method by using a leading underscore? `_get_values`

Reply:
I suppose we could do that, but I just left it as a regular class method because Enum classes seem to work a little differently than normal classes.  

For instance, if you make a normal method with the `self` parameter, it's referring to the Enum member rather than the class instance.  Also, the docs say that names that _start and end_ with a single underscore (e.g., `_ignore_`) are reserved by enum and can't be used.  I don't _think_ changing it to `_get_values` would cause an issue, but I erred on the side of caution and wrote it as a normal class method.  In a normal class I probably would have had a private variable called `_values` and then a getter method that returned that variable.

As a side note, I call `RoleEnum.get_values()` directly in `tests/integration/test_users.py` -- would that be proper for a private method (assuming python had _truly private_ methods)?

Reply:
> As a side note, I call RoleEnum.get_values() directly in tests/integration/test_users.py -- would that be proper for a private method (assuming python had truly private methods)?

I noticed that. The easy answer is No, The more complicated answer is 'it depends'.

Let's not make any changes here. It sounds like leaving it as is is the best route forward.